klarna-on-demand/src/main/java/com/klarna/ondemand/RegistrationActivity.java

```diff
@@ -1,12 +1,61 @@
 package com.klarna.ondemand;
 
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+
+import java.util.Map;
+
 /**
  * Responsible for registering a new user and setting his Klarna payment method.
  */
 public class RegistrationActivity extends WebViewActivity {
 
+    public static final String EXTRA_USER_TOKEN = "userToken";
```

Comment:
You can put it in WebViewActivity, because both registration and preference use it